cmd/tendermint/commands/reset_priv_validator.go

```diff
@@ -59,3 +60,13 @@ func resetFilePV(privValFile string, logger log.Logger) {
 		logger.Info("Generated PrivValidator", "file", privValFile)
 	}
 }
+
+// Deletes the addrbook.json file. It will be generated in node.NewNode, which is executed with
+// `tendermint start`.
+func resetAddrBook(addrBookFile string, logger log.Logger) {
+	if err := os.Remove(addrBookFile); err == nil {
+		logger.Info("Cleared AddrBook to an empty state", "file", addrBookFile)
```

Comment:
`Removed existing address book` is more human than `Cleared AddrBook to an empty state`